util/options.py

```diff
@@ -44,6 +44,9 @@ def get_subalign() -> int:
 def get_generated_c_premble() -> str:
     return opts.get("generated_c_preamble", '#include "common.h"')
 
+def get_generated_s_preamble() -> str:
+    return opts.get("generated_s_preamble")
```

Comment:
After this arg, you should add an empty string for if the value doesn't exist (so this method always in fact returns a string): `opts.get("generated_s_preamble", "")`

Reply:
Ah, I'm not overly familiar with python, so I wasn't sure if None would resolve to a string. I pushed the fix.